lib/data/models/stever/st_ever_details.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'st_ever_details.freezed.dart';
+
+part 'st_ever_details.g.dart';
+
+/// This is not full contract info, but only used fields here
+@freezed
+class StEverDetails with _$StEverDetails {
+  const factory StEverDetails({
+    required String stEverSupply,
+    required String totalAssets,
```

Comment:
What is this string means? Amount in string?

Reply:
-